### ISSUE TITLE
[HELP]: Is your instance coming back eventually?

### ISSUE THREAD

**Issue by garoto** [Sommerwiesel/invidious#9]
It was really fast...

**Comment by Sommerwiesel** [comment]
Cheers, 
I'm currently waiting for the reply from my hosting provider as well as preparing to create a proxy server that will prevent DMCA claims from ever appearing again. Expect my instance to go online at the end of February.

**Comment by garoto** [comment]
Someone sent a DMCA request to your provider and they took your box down?

**Comment by Sommerwiesel** [comment]
Invidious is a YouTube frontend, i.e. it displays YouRube channels and videos. And some channels on YouTube post pirated stuff. I got a DMCA for such pirated content and my hosting provider threatened to shut down the server. This is why I had to stop Invidious until I explained to them, that everything that Invidious does is completely legal and DMCA claims should be taken to the actual channel on YouTube. Invidious only displays what's on YouTune and has no control over that.
